playbook-website/config/menu.yml

```diff
@@ -1,111 +1,236 @@
 kits:
-  - avatars:
-    - avatar
-    - avatar_action_button
-    - multiple_users
-    - multiple_users_stacked
-    - user
-  - background
-  - bread_crumbs
-  - buttons:
-    - button
-    - button_toolbar
-    - circle_icon_button
-  - card
-  - collapsible
-  - charts_and_graphs:
-    - bar_graph
-    - circle_chart
-    - distribution_bar
-    - gauge
-    - legend
-    - line_graph
-    - treemap_chart
-  - dialog
-  - filter
-  - fixed_confirmation_toast
-  - forms:
-    - checkbox
-    - date_picker
-    - file_upload
-    - form
-    - form_group
-    - form_pill
-    - multi_level_select
-    - passphrase
-    - phone_number_input
-    - radio
-    - rich_text_editor
-    - select
-    - selectable_card
-    - selectable_card_icon
-    - selectable_icon
-    - selectable_list
-    - text_input
-    - textarea
-    - toggle
-    - typeahead
-  - highlight
-  - icon
-  - icon_circle
-  - icon_stat_value
-  - icon_value
-  - image
-  - layouts:
-    - flex
-    - layout
-  - lightbox
-  - list
-  - loading_inline
-  - map
-  - nav
-  - tags:
-    - badge
-    - hashtag
-    - pill
-  - pagination
-  - popover
-  - progress:
-    - progress_pills
-    - progress_simple
-    - progress_step
-  - section_separator
-  - star_rating
-  - table
-  - timeline
-  - time_and_date:
-    - date
-    - date_range_inline
-    - date_range_stacked
-    - date_stacked
-    - date_time
-    - date_time_stacked
-    - date_year_stacked
-    - time
-    - time_range_inline
-    - time_stacked
-    - timestamp
-    - weekday_stacked
-  - tooltip
-  - typography:
-    - body
-    - caption
-    - detail
-    - title
-  - typography_patterns:
-    - contact
-    - currency
-    - dashboard_value
-    - home_address_street
-    - label_pill
-    - label_value
-    - message
-    - person
-    - person_contact
-    - source
-    - stat_change
-    - stat_value
-    - title_count
-    - title_detail
-  - user_badge
-  - walkthrough
+  rails:
```

Comment:
[What about a different structure to indicate platforms](https://gist.github.com/jasoncypret/9f99cf83f2d508d4744612b5bea38c05)? I'm afraid this method it will be very hard to keep the categories in sync.

Reply:
I explored something very similar with him already...
kits:
  - avatars:
      platforms: ['react', 'rails']
      components:
        - avatar
        - avatar_action_button
        - multiple_users
        - multiple_users_stacked
        - user
  - background:
      platforms: ['react']
  - bread_crumbs:
      platforms: ['rails']
  - buttons:
      platforms: ['react', 'rails']
      components:
        - button
        - button_toolbar
        - circle_icon_button
  - card:
..........

Ultimately, our current method has the following benefits, which is why we ultimately chose it:
1) clearly separated: our current approach offers a clear, top-level separation between Rails and React components, making it easy to look up components for a specific platform. (especially in his aggregate_kits method that already has been written for sorting the sidebar)
2) simplicity: It's straightforward and doesn't require nesting keys like platforms under each kit, which can be simpler to parse. (visually for our sidebar and programmatically in Nick's current method)
3) reduced redundancy: If a kit is exclusive to one platform, we don't have to explicitly label it as such—it's obvious by its placement... so one less thing to do as well.

Reply:
Can you explain why this method it will be very hard to keep the categories in sync? Nick already has it working well. And both our structure and the one you proposed would use similar syntax to update and change (make new additions to) the menu.yml file with the generator.